docker-images/pysyft-worker/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# Better to not run the worker server without ID
+if [ -z "${WORKER_ID}" ]; then
+  echo "[-] You need to set the WORKER_ID environment variable"
+  exit
+fi
+
+python worker-server.py --id $WORKER_ID --port 8777
```

Comment:
Nice work. 
The port can stay fixed, as we can modify the port exposed by docker, right? (e.g. docker run -p 8770:8777 etc.)
As a next step it would be interesting to preset datasets on the websocket server worker. But this is out of the scope of this pull request.

Reply:
yeah, you can expose any available port using -p exposed_port:8777.

I will be making an example on how to run workers using a docker-compose file next. 

And yeah, the idea of the dataset is awesome, making this image a fully ready to use worker

Reply:
> I will be making an example on how to run workers using a docker-compose file next.

Can you make this in this PR? I think it would be very valuable. The instructions could be a new section in the CONTRIBUTING.md file

Reply:
I was making some examples on a separate draft PR, but yeah, I will merge that work here.